extensions/smallrye-graphql-client/deployment/src/test/java/io/quarkus/smallrye/graphql/client/deployment/DynamicGraphQLClientInjectionTest.java

```diff
@@ -38,7 +38,7 @@
                     .addAsManifestResource(EmptyAsset.INSTANCE, "beans.xml"));
 
     @Inject
-    @NamedClient("people")
+    @GraphQLClient("people")
```

Comment:
I think you also need to update the documentation: https://github.com/quarkusio/quarkus/blob/main/docs/src/main/asciidoc/smallrye-graphql-client.adoc#using-the-dynamic-client .

Reply:
Yes thanks, I missed that. Fixed now.